### ISSUE TITLE
wrap_content not working for IconTextView on v2.0.7

### ISSUE THREAD

**Issue by jhavatar** [JoanZapata/android-iconify#110]
wrap_content not working for IconTextView on v2.0.7. Works fine in v2.0.5.

**Comment by JoanZapata** [comment]
It seems to be working for me, can you show me some code showing that it's not working?

**Comment by jhavatar** [comment]
Sure. 

The chevron icon does not appear when  I display the following xml layout:
```
<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
                android:layout_width="match_parent"
                android:layout_height="wrap_content"
                android:background="#FF00FF00"
                android:padding="8dp">

    <TextView
        android:layout_width="wrap_content"
        android:layout_height="wrap_content"
        android:layout_centerVertical="true"
        android:layout_alignParentLeft="true"
        android:text="Set address"
        android:textColor="#FF000000"
        android:textSize="16sp"/>

    <com.joanzapata.iconify.widget.IconTextView
        android:text="{md-chevron-right}"
        android:textSize="32dp"
        android:textColor="#FF666666"
        android:layout_width="wrap_content"
        android:layout_height="wrap_content"
        android:layout_centerVertical="true"
        android:layout_alignParentRight="true"/>

</RelativeLayout>
```


The chevron icon does appear, but is not vertically centered as one would expect, when I  change the IcontextView's layout_width and layout_height from wrap_content to 32dp and set gravity to center:

```
<com.joanzapata.iconify.widget.IconTextView
        android:text="{md-chevron-right}"
        android:textSize="32dp"
        android:textColor="#FF666666"
        android:layout_width="32dp"
        android:layout_height="32dp"
        android:gravity="center"
        android:layout_centerVertical="true"
        android:layout_alignParentRight="true"/>
```

**Comment by JoanZapata** [comment]
Thanks for the code, I'll fix it ASAP.

**Comment by JoanZapata** [comment]
Fixed on 2.0.10, release soon.
